service/src/test/java/uk/gov/hmcts/reform/fpl/service/DocmosisTemplateDataGenerationTest.java

```diff
@@ -27,4 +27,13 @@ void shouldPopulateAMapWithWaterMarkEncoding() throws IOException {
             .asString()
             .isNotEmpty();
     }
+
+    @Test
+    void shouldPopulateAMapWithCourtSealEncoding() throws IOException {
+        final Map<String, Object> draftWaterMarkData = templateDataGeneration.getCourtSealData();
+        assertThat(draftWaterMarkData).containsKey("courtseal")
+            .extracting("courtseal")
+            .asString()
```

Comment:
is the asString necessary? can we not make an assertion straight on what you have extracted?

Reply:
good point, not needed and removed :)